ptsa/data/timeseries.py

```diff
@@ -322,12 +322,17 @@ def _from_hdf_legacy(cls, filename):
     def append(self, other, dim=None):
         """Append another :class:`TimeSeries` to this one.
 
+        .. versionchanged:: 2.0
+           Appending along a dimension not present will cause that
+           dimension to be created.
+
         Parameters
         ----------
         other : TimeSeries
         dim : str or None
             Dimension to concatenate on. If None, attempt to concatenate all
-            data (likely to fail with truly multidimensional data).
+            data (likely to fail with truly multidimensional data). If not present,
```

Comment:
Also, we should add a test case for the functionality of adding a new dimension rather than only relying on testing this indirectly through the tests for MorletWaveletFilter.

Reply:
@leondavis1 can you clarify what the comment means?